.github/workflows/release_build.yml

```diff
@@ -120,6 +120,14 @@ jobs:
           tags: |
             ${{ env.RELEASE_PRIVATE_REPOSITORY }}:v${{ github.event.inputs.version }}
 
+      - name: Get SHA256 checksum of wheel file
+        id: get_sha256
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          filename="aws_opentelemetry_distro-${{ github.event.inputs.version }}-py3-none-any.whl"
```

Comment:
nit: `filename` is used in more than one place, could be worth setting it to an environment variable for reuse to avoid any mistakes in the future